### PR TITLE
Add Gherkin stories for editing and previewing lessons/quizzes

### PR DESCRIPTION
**Pull request by khangmach**
Adds stories for the following:
- Edit lesson from reports tab
- Edit quiz from reports tab
- Preview quiz
- Preview lesson

### CODE REVIEW

File: integration_testing/features/coach/coach-edit-lesson-from-report.feature [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  Class coaches and normal coaches are able to preview quizzes they create in the plan tab
+
+  Background:
+    Given I have finished creating a '<lesson>' Lesson A
+      And I am signed in to Kolibri as a <class coach> or <coach>
+      And the lesson can be inactive or active and assigned to anyone
+
+  Scenario: Edit lesson details from its report page
+    When as as a coach I go to Coach and click into the 'Reports' tab
+    Then I click into Lesson A report
```
Comment (jonboiser) [inline integration_testing/features/coach/coach-edit-lesson-from-report.feature]: Combine the above "When" and "Then" into the "Given", so the starting point for the test is being on the Report page. e.g. "And I am on the report page for Lesson A"

Comment (jonboiser) [reply integration_testing/features/coach/coach-edit-lesson-from-report.feature]: Apply this to the other scenarios, so there is a shorter path to the Feature we're trying to test.

**Comment by radinamatic** [comment]
Seemed easier to [open a PR](https://github.com/khangmach/kolibri/pull/1) on top of your PR :slightly_smiling_face: 
Feel free to merge and add more changes if necessary!

**Review by radinamatic — APPROVED** [review]
This looks good to me! :+1: 
Let's merge, and I'll make another pass through all the gherkins (not just coach) before the next integration testing round :slightly_smiling_face:

**Review by jonboiser — CHANGES_REQUESTED** [review]
I have another PR on khangmach/gherkins to shorten the scenarios and add other details that should be testd

**Comment by radinamatic** [comment]
Awesome, thanks @jonboiser :+1:

**Review by jonboiser — APPROVED** [review]

**Comment by indirectlylit** [comment]
Thanks all - great job!

Moving forward I'd like to use this iterative approach to the stories in git from the start: @khangmach and @jtamiace can submit the skeletons early, and the rest of us can give feedback and then make updates to them

**Comment by radinamatic** [comment]
Seems like a great workflow to use! Yey teamwork :busts_in_silhouette:

**Pull request merged**